### ISSUE TITLE
Yarn Build Failing - ignoring Token Entangler & Candy Machine UI

### ISSUE THREAD

**Issue by JYeswak** [metaplex-foundation/metaplex#1541]
<img width="505" alt="Screen Shot 2022-01-13 at 9 47 09 AM" src="https://user-images.githubusercontent.com/87095688/149380771-5dd2b41d-e533-443d-8a76-e2bf84866d59.png">
<img width="610" alt="Screen Shot 2022-01-13 at 10 22 50 AM" src="https://user-images.githubusercontent.com/87095688/149380774-26ad6706-b67b-4b46-9d01-38c93b04afd8.png">

I am trying to run command NPX Yarn Build and it keeps failing 

MacBook-Pro-Josh:js josh$ npx yarn build
yarn run v1.22.17
**$ lerna run build --ignore=token-entangler --ignore=candy-machine-ui**
lerna notice cli v3.22.1
lerna info versioning independent
**lerna notice filter excluding ["token-entangler","candy-machine-ui"]**
lerna info filter [ '!token-entangler', '!candy-machine-ui' ]
lerna info Executing command in 4 packages: "yarn run build"
lerna info run Ran npm script 'build' in '@metaplex/cli' in 7.1s:
$ tsc -p ./src

The terminal stops at this point and won't continue.

**Comment by JYeswak** [comment]
I ended up getting past this point; I realized I had not properly installed TS Node in a previous step. I had to install it with a sudo command.

**Comment by tomtom1233** [comment]
Can you please tell me what command you have used to make it work. I currently have the same problem

**Comment by JYeswak** [comment]
I never fixed where it says it was excluded - it remained excluded through the process.  I also had several errors in my build response and just moved forward with the next steps.  

that said - the problem you are having doesn't match what was happening to me - where it says **$ tsc -p ./src**, mine would freeze and not continue.  I ended up installing ts-node with a sudo command, as originally I kept getting permission errors.  

Admittedly, I was trying so many things back to back I don't remember exactly what I did. Not a very helpful response, I know...
